src/app/my-scs/components/sc-card/sc-card.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'app-sc-card',
+  template: `
+    <div class="card-container" routerLink={{card.scId}} routerLinkActive="active">
+      <mat-card class="card">
+        <div class="card-header">
+          <mat-card-title>
+            <div><p>Bewertungszeitraum</p>
```

Comment:
bewertungszeitraum should come from de.json
`<p>{{'dashboard.timerange' | translate}}</p>`